msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/crosscloud/TestCase1592468.java

```diff
@@ -121,17 +119,10 @@ public void test_1592468() throws Throwable {
 
         // Acquire token interactively from cross cloud
         final MsalAuthResult acquireTokenCrossCloudResult = msalSdk.acquireTokenInteractive(acquireTokenCrossCloudAuthParams, crossCloudInteractionHandler, TokenRequestTimeout.MEDIUM);
-        Assert.assertFalse("Verify accessToken is not empty", TextUtils.isEmpty(acquireTokenCrossCloudResult.getAccessToken()));
+        Assert.assertFalse("AccessToken is empty", TextUtils.isEmpty(acquireTokenCrossCloudResult.getAccessToken()));
 
-        JSONObject profileObject = getProfileObjectFromMSGraph(acquireTokenCrossCloudResult.getAccessToken());
-        Assert.assertEquals("Verify email returned from graph call matches the username", userName, profileObject.get("mail"));
-
-        // Verify the account object contains tenant profile corresponding to the cross cloud guest account
-        MultiTenantAccount account = (MultiTenantAccount) msalSdk.getAccount(mActivity, getConfigFileResourceId(), userName);
-        ITenantProfile tenantProfile = account.getTenantProfiles().get(mGuestUser.getGuestLabTenants().get(0));
-        Assert.assertNotNull(tenantProfile);
-        Assert.assertTrue(tenantProfile.getClaims().get("iss").toString().contains(mCrossCloud));
-        Assert.assertTrue(tenantProfile.getClaims().get("idp").toString().contains(mHomeCloud));
```

Comment:
Why removing this verification

Reply:
This test case was a bit inaccurate from the ado item. There was no explicit requirement for this check, so I removed it.